conf/application.conf

```diff
@@ -17,10 +17,13 @@ play.modules.enabled += "inject.ApplicationProductionModule"
 # The application configuration
 # ~~~~~
 application.settingsWidget.enabled = true
-
-# overridden configuration from CTP project, default first
-#application.countries = ["DE", "US", "GB", "AT", "CH"]
-#application.languages = ["en", "de", "es"]
+#application.countries = ["DE", "US", "GB", "AT", "CH"] # uncomment to override countries from CTP project, default first
+#application.i18n.languages = ["en", "de", "es"] # uncomment to override languages from CTP project, default first
```

Comment:
Yes, of course. It is just reading and parsing them into locales. Then the usage you do with those locales it's up to each particular situation. Exactly the same would happen if those locales are coming from CTP.